src/main/java/edu/greenblitz/utils/InterruptableWaitCommand.java

```diff
@@ -0,0 +1,29 @@
+package edu.greenblitz.utils;
+
+import edu.wpi.first.wpilibj.Timer;
+
+public class InterruptableWaitCommand extends GBCommand{
```

Comment:
Maybe I don't remember correctly, but I belive you me and Tal talked about this, and why we don't really need this class.